test-packages/integration-tests/test/auth-flows/auth-flow.spec.ts

```diff
@@ -51,7 +63,7 @@ describe('OAuth flows', () => {
     const result = await BusinessPartner.requestBuilder()
       .getAll()
       .top(1)
-      .execute(destination!);
+      .execute(destination! as any);
```

Comment:
Could you please share why `as any` is needed here?

Reply:
I added a TODO in the beginning of the file. The reason is that we install the business partner vdm, which is based on core 1.53. On the other had I want to use the `getDstination` from the local sources to debug i.e. `connectivity` varion 2.0.
I updated the auth types of the destination which makes the types used by the `getDestination` and `execute` incompatible. This will disappear once we updated the VDM.

Reply:
This means, we test 1.53.x client with 2.0 core modules, which might be tricky.
As this is only the case for this file, it's still fine.